### ISSUE TITLE
Message editor / broadcasting

### ISSUE THREAD

**Issue by kmeesters** [FirstLegoLeague/displaySystem#26]
Not sure if this should be part of the display system, but some type of messages could be used elsewhere. For example a public announcement. A quick fix could be to mark a new lower third as a 'public notification', which could be picked up by other application (for example team app)

**Comment by Jheronymus** [comment]
maybe a ticker in the top of the screen?... on TV this is usually the 'element'  they use to display 'out of scope'  information, like schedule changes...

**Comment by kmeesters** [comment]
Yes, the question is should the display system be the input for this? Maybe need to split at some point between a director-app and the display. I'd indeed say a ticker would be great (see EAS): https://en.wikipedia.org/wiki/Emergency_Alert_System ;) https://youtu.be/VLg_MKGFDrc?t=1m12s

**Comment by rikkertkoppes** [comment]
The ticker is a perfect match for this. However, there can currently be only one instance of a module, so i would suggest. 
- alowing to create multiple instances of the same module, giving a class or idto style them differently
- decoupling the mhub node and topic they listen to
- loosening up styling, allowing easier overrides an tighter internals.

Message input is a bit out of scopn for display system, but the config panel could be a rudimentary ui for this
